src/parse.rs

```diff
@@ -474,6 +491,25 @@ impl Extensions {
     }
 }
 
+pub trait Num: Sized {
+    fn from_str(src: &str, radix: u32) -> ::std::result::Result<Self, ()>;
```

Comment:
since our `Result` collides with standard one, can we `use std::result::Result as StdResult`?